pgmoon/init.lua

```diff
@@ -687,7 +687,8 @@ do
         self.luasec_opts = {
           key = opts.key,
           cert = opts.cert,
-          cafile = opts.cafile
+          cafile = opts.cafile,
+          ssl_version = opts.ssl_version
```

Comment:
I am assuming you manually verified this? I don't think writing a test for this one off makes the most sense and based on previous commits for SSL additions this looks correct.

Reply:
Indeed, if we don't do this, the default `TLSv1.1` will be used by the driver: https://github.com/leafo/pgmoon/blob/v1.11.0/pgmoon/socket.lua#L52

Which won't work if we set `ssl_min_protocol_version` to `TLSv1.2` in the postgres settings